pages/trust-admin.js

```diff
@@ -18,9 +18,11 @@ const TrustAdmin = ({
   leastVisited,
   mostVisited,
   trust,
+  averageParticipantsInVisit,
+  averageParticipantsInVisitError,
   visitsScheduled,
 }) => {
-  if (wardError || trustError) {
+  if (wardError || trustError || averageParticipantsInVisitError) {
     return <Error />;
```

Comment:
Is it time to consider grouping up the errors so that we can pass them through to the Error component? That way the error will be sent to Sentry and we will be notified

Reply:
Not 100% sure what you mean but had a look at how it's used elsewhere. I think you mean passing down these errors into the `<Error err={error}` component. How do you imagine them being grouped? 🤔

Reply:
Yes sorry for being vague, I meant doing `<Error err={error}>`. It only really takes one exception object so since we have 3 we could pass them all in and let it handle returning the first one that is not null (`<Error err={wardError || trustError || averageParticipantsInVisitError}>`

Reply:
No worries! That makes sense, will update.